doc/api/buffer.markdown

```diff
@@ -468,7 +468,7 @@ additional performance that `Buffer.allocUnsafe(size)` provides.
 
 ### Class Method: Buffer.byteLength(string[, encoding])
 
-* `string` {String}
+* `string` {String|Buffer|DataView|ArrayBuffer}
```

Comment:
I ask this b/c everything listed here is a discrete constructor. Which at quick glance would show that all typed arrays are not supported.